.claude/skills/authentication/reference/examples/auth-patterns.py

```
@@ -0,0 +1,886 @@
+
+    def create_token(self, user, request, device_name):
+        """Create token with device tracking."""
+        # Try to use ManagedToken if available
+        try:
+            token = ManagedToken.objects.create(
+                user=user,
+                device_name=device_name,
+                ip_address=ManagedToken.get_client_ip(request),
+                user_agent=request.META.get('HTTP_USER_AGENT', '')[:500],
+            )
+        except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)